staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -793,7 +793,6 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 
 		columns, err := getColumnsForVersion(crd, v.Name)
 		if err != nil {
-			utilruntime.HandleError(err)
```

Comment:
I think this is probably OK (CRD authors won't be looking in apiserver logs for this error!), but it'd be better if we returned a better error indicating the CRD didn't support the format or something. I guess I won't ask for you to fix that since it is pre-existing.

Reply:
I'm happy to send a follow up PR here, we probably should also wrap the error here as well like you mentioned :D